apps/test-viewer/src/components/quantity-formatting/FormatSetsTabPanel.tsx

```diff
@@ -0,0 +1,185 @@
+
+// Test format sets for development
+const testFormatSets: FormatSet[] = [
+  {
+    name: "TestFormatSet1",
+    label: "Arizona Highway Project (Civil)",
+    unitSystem: "imperial",
+    description:
+      "This format set contains all the formatting standards used by civil engineers on the Arizona Highway Project. Includes units for measurements, coordinates, and construction materials.",
+    formats: {},
+  },
+  {
+    name: "TestFormatSet2",
+    label: "Arizona Highway Project (Project Manager)",
+    unitSystem: "imperial",
+    description: "This format set contains all the formatting standards used by project managers on the Arizona Highway Project.",
+    formats: {},
+  },
+  {
+    name: "TestFormatSet3",
+    label: "My personal format set",
+    unitSystem: "metric",
+    description: "Custom formatting preferences for individual use. Combines metric and imperial units based on personal workflow requirements.",
+    formats: {},
+  },
+];
+
+/** Tab panel component for Format Sets management */
+export const FormatSetsTabPanel: React.FC = () => {
+  const [formatSets, setFormatSets] = React.useState<FormatSet[]>([]);
+  const [activeFormatSetKey, setActiveFormatSetKey] = React.useState<string | undefined>();
+  const [selectedFormatSetKey, setSelectedFormatSetKey] = React.useState<string | undefined>();
+  const [selectedFormatSet, setSelectedFormatSet] = React.useState<FormatSet | undefined>();
+  const [clonedSelectedFormatSet, setClonedSelectedFormatSet] = React.useState<FormatSet | undefined>();
+  const [saveEnabled, setSaveEnabled] = React.useState(false);
+
+  // Load format sets from FormatManager
+  React.useEffect(() => {
+    const formatManager = FormatManager.instance;
+    if (formatManager) {
+      // Combine FormatManager format sets with test format sets
+      const combinedFormatSets = [...formatManager.formatSets, ...testFormatSets];
+      setFormatSets(combinedFormatSets);
+
+      // Set active format set if one exists
+      if (formatManager.activeFormatSet) {
+        const activeKey = formatManager.activeFormatSet.name;
+        setActiveFormatSetKey(activeKey);
+        setSelectedFormatSetKey(activeKey); // Initially select the active one
+        setSelectedFormatSet(formatManager.activeFormatSet);
+        setClonedSelectedFormatSet({ ...formatManager.activeFormatSet });
+      }
+
+      // Listen for active format set changes
+      const removeListener = formatManager.onActiveFormatSetChanged.addListener((formatSet) => {
+        if (formatSet) {
+          setActiveFormatSetKey(formatSet.name);
+        } else {
+          setActiveFormatSetKey(undefined);
+        }
+      });
+
+      return () => {
+        removeListener();
+      };
+    } else {
+      // If no FormatManager, just use test format sets
+      setFormatSets(testFormatSets);
+    }
+  }, []);
+
+  const handleFormatSetChange = React.useCallback((formatSet: FormatSet, key: string) => {
```

Comment:
Maybe rename this to `handleSelectedFortmatSetChange`. As it is very confusing having `handleFormatSetChange` and `handleFormatSetChanged`.